Sources/Algorithms/Unique.swift

```diff
@@ -42,14 +42,14 @@ extension Sequence {
   /// by the given projection), in the order of the first occurrence of each
   /// unique element.
   ///
-  ///     let animals = ["dog", "pig", "cat", "ox", "dog","cat"]
-  ///     let uniqued = animals(on: { $0.first })
+  ///     let animals = ["dog", "pig", "cat", "ox", "dog", "cat"]
```

Comment:
This example doesn't really demonstrate the role of the 'projection' closure. If you want to stick to the projection `$0.first`, I'd suggest something like:

```suggestion
  ///     let animals = ["dog", "pig", "cat", "ox", "cow", "owl"]
```